platform-sdk/swirlds-merkle/src/main/java/com/swirlds/merkle/map/MerkleMapMetrics.java

```diff
@@ -64,7 +62,7 @@ public final class MerkleMapMetrics {
     private MerkleMapMetrics() {}
 
     /**
-     * Gets a value indicating whether the {@link SwirldMain} has called the {@link
+     * Gets a value indicating whether the SwirldMain has called the {@link
```

Comment:
The `@link` is missing

Reply:
This link can no longer exist without adding a circular dependency on swirlds common and swirlds core.